exporter/loggingexporter/factory.go

```diff
@@ -43,28 +44,53 @@ func (f *Factory) CreateDefaultConfig() configmodels.Exporter {
 			TypeVal: typeStr,
 			NameVal: typeStr,
 		},
+		LogLevel: "info",
 	}
 }
 
-func noopStopFunc() error {
-	return nil
-}
-
 // CreateTraceExporter creates a trace exporter based on this config.
-func (f *Factory) CreateTraceExporter(logger *zap.Logger, cfg configmodels.Exporter) (consumer.TraceConsumer, exporter.StopFunc, error) {
+func (f *Factory) CreateTraceExporter(logger *zap.Logger, config configmodels.Exporter) (consumer.TraceConsumer, exporter.StopFunc, error) {
+	cfg := config.(*Config)
 
-	lexp, err := NewTraceExporter(cfg.Name(), logger)
+	le, err := f.createLogger(cfg.LogLevel)
```

Comment:
nit: `logger` instead of `le`

Reply:
Thanks for pointing this out. But the variable name logger has been used by the method parameter. Instead, `le` seems to be consistent with the names used in other exporter files like `oce`, `ze`. Do you have any suggestion about the variable and the parameter names?

Reply:
Hum `le` is not an exporter, it is a logger that's why I don't like `le`. Perhaps a full name like  `exporterLogger` will make it clear. Side note: you could use `logger` but I agree that it may be too subtle and get readers confused.

Reply:
Right, `exporterLogger` is better than `le`. Please review the latest commit. Thanks.